docs.feldera.com/docs/connectors/sources/delta.md

```diff
@@ -192,6 +193,38 @@ The following table lists supported Delta Lake data types and corresponding Feld
 | `VARIANT`                   | `VARIANT`        |               |
 
 
+## Transactions
+
+The Delta Lake connector can be configured to automatically initiate [transactions](/pipelines/transactions)
+when ingesting the table snapshot and following the transaction log.
+The `transaction_mode` property configures this feature:
+
+* `none` - the connector does not group inputs into transactions.
+* `snapshot` - ingest the initial snapshot of the table in one or several transactions.
+* `always` - all updates generated by the connector are grouped into transactions, both when reading the initial snapshot
+  and when following the transaction log.
+
+### Ingesting table snapshot using transactions
+
+If the connector is configured in the `snapshot` or `snapshot_and_follow` mode, and its
+`transaction_mode` is set to `snapshot` or `always`, it will ingest the snapshot of the
+table in one or several transactions. The exact behavior depends on the value of the `timestamp_column` option.
+If `timestamp_column` is not set, the connector will ingest the snapshot of the table in one big transaction.
+
+If `timestamp_column` is set, the connector will ingest the snapshot of the table in a series of batches, one for
+each timestamp range of width equal to the `LATENESS` attribute of the `timestamp_column`. Each range will be
+ingested in a separate transaction.
+
+See `timestamp_column` documentation for more details.
+
+### Ingesting the Delta transaction log using transactions
+
+If the connector is configured in the `follow`, `snapshot_and_follow`, or `cdc` mode, and its
+`transaction_mode` is set to `always`, it will ingest the transaction log of the table in a series of transactions,
+generating exactly one transaction for each entry in the table's transaction log.
+
+In other words, Feldera transaction boundaries will precisely match the transaction boundaries of the Delta Lake table.
```

Comment:
and this never ends, right?
as long as the table keeps changing, each log segment (or entry?) will initiate a new transaction.